### PR TITLE
Doxygen Documentation: stdint.h Fixes #25

### LINKED ISSUE #25: Doxygen Documentation: stdint.h

### PR DESCRIPTION
**Pull request by seekaddo**
I think this is ready now.

I am trying to figure out more detailed information for `__WORDSIZE`

### CODE REVIEW

**Comment by seekaddo** [comment]
@phillipjohnston Are this reflags normal?

File: include/stdint.h [hunk 1]
```diff
@@ -9,8 +9,12 @@
 #include <limits.h>
 
 #if __LP64__
+
+/*! Expands to 64. System word size */
 #define __WORDSIZE 64
```
Comment (phillipjohnston) [inline include/stdint.h]: These are internal values that are used for selected some definitions later (e.g. uintptr max)

**Comment by seekaddo** [comment]
@phillipjohnston What do you mean specifically?

> These are internal values that are used for selected some definitions later (e.g. uintptr max)

I know that they are values for the compiler. But The information given is not enough. I wanted something more clear for the developer.

**Comment by seekaddo** [comment]
@phillipjohnston oh sorry. I got you now. I fixed it.

File: include/stdint.h [hunk 2]
```diff
@@ -168,34 +296,66 @@ typedef uint64_t uint_fast64_t;
    the type. */
 #ifndef WCHAR_MIN
 #if WCHAR_MAX == 0xffff
+
+/*! Exapnds to 0. Minimum value of object of wchar_t type */
```
Comment (phillipjohnston) [inline include/stdint.h]: Typo: Exapnds

File: include/stdint.h [hunk 3]
```diff
@@ -168,34 +296,66 @@ typedef uint64_t uint_fast64_t;
...
 #define WCHAR_MIN 0
 #else
+
+/*! Exapnds to (-WCHAR_MAX - 1). Minimum value of object of wchar_t type */
```
Comment (phillipjohnston) [inline include/stdint.h]: Typo: Exapnds

File: include/stdint.h [hunk 4]
```diff
@@ -60,104 +92,200 @@ typedef uint64_t uint_fast64_t;
...
 #endif
 
 #ifndef WCHAR_MAX
 #ifdef __WCHAR_MAX__
+
+/*! Maximum value of object of wchar_t type.  */
 #define WCHAR_MAX __WCHAR_MAX__
 #else
+
+/*! Exapnds to 0x7fffffff.  Maximum value of object of wchar_t type  */
```
Comment (phillipjohnston) [inline include/stdint.h]: Typo: Exapnds

File: include/stdint.h (hunk 3 above)
Comment (seekaddo) [reply include/stdint.h]: Thanks for the review. I will fix this.

**Comment by seekaddo** [comment]
@phillipjohnston I think it looks good now.

**Pull request merged**